hapi-fees/src/main/java/com/hedera/services/pricing/BaseOperationUsage.java

```diff
@@ -219,6 +227,22 @@ UsageAccumulator fileAppend() {
 		return into;
 	}
 
+
+	UsageAccumulator tokenFreezeAccount() {
+		final var tokenFreezeMeta = TOKEN_OPS_USAGE_UTILS.tokenFreezeUsageFrom();
+		final var into = new UsageAccumulator();
+		TOKEN_OPS_USAGE.tokenFreezeUsage(SINGLE_SIG_USAGE, NO_MEMO_AND_NO_EXPLICIT_XFERS, tokenFreezeMeta, into);
+		log.info("TokenFreeze base accumulator: {}", into);
```

Comment:
Intriguing...do you know a way to enable logging when this method runs from JUnit? I would happy to know that. 😁

Reply:
Unfortunately I haven't tried that.  I should remove (and already did in another PR) this line. 
I guess we need use some mechanism like log captor t o do so. It's a good practice to make this work inside Junit run though.